.github/workflows/e2e.yaml

```diff
@@ -2,44 +2,42 @@ name: End-to-End Tests
 on:
   workflow_dispatch:
   pull_request:
-concurrency: build-wsl
+concurrency: e2e

```

Comment:
Shouldn’t that be "end-to-end-tests"?

Reply:
You can actually use any string here. From the docs:
> Use concurrency to ensure that only a single job or workflow using the same concurrency group will run at a time. A concurrency group can be any string or expression.
[...]
When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be pending.

I still changed it to your sugestion because I like that name better 😅.